flink-test-utils-parent/flink-migration-test-utils/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.flink</groupId>
+        <artifactId>flink-test-utils-parent</artifactId>
+        <version>1.17-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>fink-migration-test-utils</artifactId>
```

Comment:
The initial though is that the module needs to have minimum number of dependences since it needs to be dependent by other modules. `flink-test-utils` is not suitable since it already depends on `flink-core`, `flink-runtime`, etc., which would cause circular dependency problem. `flink-test-utils-junit` might work for now, but I'm not pretty sure if it will also introduce other dependencies in the future.